src/Model/Item.php

```diff
@@ -156,10 +178,7 @@ public static function delete($item_id, $priority = PRIORITY_HIGH)
 
 		// If it's the parent of a comment thread, kill all the kids
 		if ($item['id'] == $item['parent']) {
-			$items = dba::select('item', ['id'], ['parent' => $item['parent']]);
-			while ($row = dba::fetch($items)) {
-				self::delete($row['id'], $priority);
-			}
+			self::delete(['parent' => $item['parent']]);
```

Comment:
Why not keep the priority in the `self::delete` call?

Reply:
Because I forgot.